packages/backend/src/utils/vscodeProxy.ts

```diff
@@ -0,0 +1,79 @@
+import { set } from "lodash";
+import { join } from "path";
+
+export const getVscode = () => {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    return require("vscode");
+  } catch (error) {
+    return undefined;
```

Comment:
redundant

Reply:
Do you suggest to leave catch block empty ?